.github/workflows/deploy-earn-protocol-landing-page-staging.yaml

```
@@ -0,0 +1,125 @@
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build docker image, copy build output and push to ECR
+        id: build-image
+        env:
+          LATEST_TAG: latest
+          ECR_REPO_NAME: summer-fi-earn-protocol-landing-page-staging
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          SHA_TAG: ${{ steps.vars.outputs.sha_short }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:9:17: Double quote to prevent globbing and word splitting

(shellcheck)